### PR TITLE
site: refactor CC => FC 

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | site: refactor CC => FC  |
| 🇨🇳 Chinese | site: refactor CC => FC  |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .dumi/theme/builtins/Previewer/index.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import type { Project } from '@stackblitz/sdk';
 import { Alert, Badge, Tooltip, Space } from 'antd';
 import classNames from 'classnames';
 import LZString from 'lz-string';
-import React from 'react';
+import React, { useContext, useEffect, useRef, useState } from 'react';
```
Comment (afc163) [inline .dumi/theme/builtins/Previewer/index.tsx:7]: .dumi 下有一堆 ts 报错，但现在 npm run tsc 检查不出来。最好一起看看。

Comment (afc163) [reply .dumi/theme/builtins/Previewer/index.tsx:7]: 图片

可以加个这个

Comment (afc163) [reply .dumi/theme/builtins/Previewer/index.tsx:7]: 图片

Comment (li-jia-nan) [reply .dumi/theme/builtins/Previewer/index.tsx:7]: image

问题有点多，我新开一个 pr 专门搞 ts 问题吧，这里不改了

**Review by afc163 — APPROVED** [review]

**Review by afc163 — APPROVED** [review]

**Pull request merged**